src/pages/workspace/accounting/qbo/import/QuickbooksImportPage.tsx

```diff
@@ -51,7 +51,7 @@ function QuickbooksImportPage({policy}: WithPolicyProps) {
         },
     ];
 
-    if (policy?.connections?.quickbooksOnline.data.country !== CONST.COUNTRY.US) {
+    if (policy?.connections?.quickbooksOnline?.data?.country !== CONST.COUNTRY.US) {
```

Comment:
should we also mark these fields optional in the TS definition (Policy.ts)?

<img width="500" alt="Screenshot 2024-05-02 at 1 08 04 PM" src="https://github.com/Expensify/App/assets/98560306/fe6d42e3-20d1-4932-aa84-29c461ab414c">

Reply:
hmm that is probably a good idea, then we wouldn't get a random crash because our types would catch it

Reply:
I tried it, but it breaks all calls to `updatePolicyConnectionConfig` because something goes wrong with the parameter types:

<img width="783" alt="image" src="https://github.com/Expensify/App/assets/87341702/0d4ffecf-3ab1-4ef0-9d2d-d0357eb86046">

I'll leave investigating this for another occasion